ts/src/flexible-event/privacy.ts

```diff
@@ -148,6 +163,17 @@ function capacityQarySymmetricChannel(
   )
 }
 
+function attributionScopesInformationGain(
+  numStates: number,
+  attributionScopeLimit: number | null,
```

Comment:
Could this be null if it's only called when the scope limit is not null?

Reply:
Removed `| null`.